stencils/pace/stencils/testing/test_translate.py

```diff
@@ -183,11 +183,22 @@ def perturb(input):
 
 
 def get_thresholds(testobj, input_data):
+    return _get_thresholds(testobj.compute, input_data)
+
+
+def get_thresholds_parallel(testobj, input_data, communicator):
+    def compute(input):
+        return testobj.compute_parallel(input, communicator)
+
+    return _get_thresholds(compute, input_data)
```

Comment:
This unused function can safely be committed, I used it manually to get thresholds on the parallel translate test.